tests/unit/db/account_tests.py

```diff
@@ -232,14 +234,14 @@ def test_delete_remote_db_via_delitem(self):
         dbname = self.dbname()
         try:
             self.client.connect()
-            db = self.client.create_database(dbname)
+            _ = self.client.create_database(dbname)
             self.assertIsNotNone(self.client.get(dbname))
             self.client.__delitem__(dbname, remote=True)
             # Removed from local cache
             self.assertIsNone(self.client.get(dbname))
             # Database removed remotely as well
             try:
-                db = self.client[dbname]
+                _ = self.client[dbname]
```

Comment:
We've been a bit more lax with the test code as far as standards go.  I had left the `db` in there for it to be a bit clearer as to the purpose of the test.  

I guess my main concern initially was that some changes that were being made in this PR did not seem like they were py2 to py3 related. After reading your responses to some of the earlier comments I realize now that that is not the case. If you ran pycharm on the entire testing code base then obviously you made this type of "fix" everywhere pycharm had a similar issue so I am fine with you leaving this as it is throughout.